index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="top">
+        <h2 class="logo">FABIEN</h2>
+        <button class="ham" type="button">
+          <img src="./img/Union.png" alt="hamburger img" />
+        </button>
+      </header>
+      <section class="headline">
+        <div class="name">
+          <h1 class="fab">Fabien</h1>
+        </div>
+        <div class="dev">
+          <h2 class="soft">Software Developer</h2>
+        </div>
+        <div class="para">
+          <p class="pa">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+      </section>
+      <img src="./img/Devider.png" alt="" />
+      <section class="bottom">
+        <div class="lang">
+          <h4 class="main">Languages</h4>
+          <h4>Javascript</h4>
+          <h4>Ruby</h4>
+          <h4>Html</h4>
```

Comment:
- [ ] Instead of using the `section` tag on line 34, It would be better to use a `div` tag. Then, wrap all your code from line 33 up to 72 in the headline section( section with class="headline"), this will make your code clean, and like that you will have semantic HTML tags.